src/components/organisms/UiPhoneNumber/_internal/UiPhoneNumberPrefix.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <UiDropdown
+    class="ui-phone-number__dropdown"
+    :model-value="modelValue"
+    :popover-attrs="{ class: 'ui-phone-number__dropdown-popover' }"
+    @update:model-value="handleOnChangeCode"
+  >
+    <template
+      #toggle="{
+        toggleHandler, isOpen,
+      }"
+    >
+      <UiButton
+        id="ui-phone-number__dropdown"
+        class="ui-button--outlined ui-button--has-icon ui-phone-number__dropdown-button"
+        :class="{ 'ui-button--has-error': error }"
+        type="button"
```

Comment:
why we need `type="button"` UiButton has `type="button"` you add it in the previous PR

Reply:
I have to merge/rebase develop